src/fides/api/models/property.py

```diff
@@ -45,6 +55,9 @@ def __tablename__(self) -> str:
         primary_key=True,
         default=generate_id,
     )
+    # Right now, we use server default to write this val.
+    # In the future we may allow ability to configure which property is the default
+    is_default = Column(Boolean, server_default="f", default=False)
```

Comment:
Refresh my memory please, is it the intent that you can have multiple MessagingTemplates of the same "type" as long as only one is enabled?